data/mods/Xedra_Evolved/items/inventor/explosive.json

```diff
@@ -0,0 +1,48 @@
+[
+  {
+    "id": "portable_dark_age",
+    "type": "TOOL",
+    "category": "weapons",
+    "name": { "str": "portable dark age" },
+    "description": "In this device the Mad Genius creates an extremely strong magnetic field.  Then in the next step of it's creation he splits that field in two in such a way it is perpetually trying to reconnect.  When activated the timer counts down til the moment the two fields reconnect.  Upon reconnection energy is released in the form of EMP.",
+    "weight": "400 g",
+    "volume": "200 ml",
+    "price": "600 USD",
+    "price_postapoc": "300 USD",
+    "to_hit": -1,
+    "material": [ "steel", "plastic" ],
+    "symbol": "0",
+    "explode_in_fire": false,
+    "color": "light_gray",
+    "use_action": {
+      "need_wielding": true,
+      "target": "portable_dark_age_apct",
+      "msg": "You activate the portable dark age.",
+      "target_timer": "50 seconds",
+      "menu_text": "Activate bomb",
+      "type": "transform"
+    },
+    "flags": [ "RADIO_MODABLE", "RADIO_INVOKE_PROC", "BOMB", "GRENADE", "WATER_BREAK" ],
```

Comment:
If I understand its purpose correctly (and assuming that it isn't EMP-proofed), the "ELECTRONIC" flag is missing. Can a "portable dark age" be broken by an EMP, and if so, is it fail-safe or fail-deadly (chain "explosion")? If it can be broken by an EMP, then carrying multiple "portable dark age" can be a bit risky (don't want to fry your other EMP bombs).

Reply:
> If I understand its purpose correctly (and assuming that it isn't EMP-proofed), the "ELECTRONIC" flag is missing. Can a "portable dark age" be broken by an EMP, and if so, is it fail-safe or fail-deadly (chain "explosion")? If it can be broken by an EMP, then carrying multiple "portable dark age" can be a bit risky (don't want to fry your other EMP bombs).

I've assumed it would be shielded because we don't have a way to chain effects off EMP destruction of electronics.